src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -332,6 +332,13 @@ private void DisplayExtensionList()
                     if(node.TargetFramework != null)
                         _outWriter.Write(ColorStyle.Value, $"(.NET {node.TargetFramework?.FrameworkVersion})");
```

Comment:
Now we're adding extra stuff to this output, I think it would be good if the assembly target framework also displayed on a new line in the same style as Version/Path? Happy for this to be a separate PR though, if you'd rather just get this merged. 🙂

Reply:
@ChrisMaddock My time is a bit short so I'll leave tuning the output format to another PR.